pwa/static/themes/rotterdam.css

```diff
@@ -182,16 +182,12 @@
   --web-app-secondary-top-nav-item-padding: var(--web-app-size-md);
   --web-app-secondary-top-nav-font-size: var(--web-app-font-size-md);
   --web-app-secondary-top-nav-color: var(--web-app-color-black);
-  --web-app-secondary-top-nav-background-color-hover: auto;
   --web-app-secondary-top-nav-font-weight: var(--web-app-font-weight-light);
 
   /* @condcution: Primary Top Navigation */
   --web-app-primary-top-nav-item-padding: var(--web-app-size-md);
   --web-app-primary-top-nav-color: var(--web-app-color-white);
   --web-app-primary-top-nav-background-color: var(--rotterdam-color-green);
-  --web-app-primary-top-nav-background-color-hover: var(
```

Comment:
You've removed rules from `rotterdam.css` and from `utrecht.css`, whenever you do this, you should also remove them from `theme.css.example` as this is the template for all other implementation themes (e.g. rotterdam and utrecht, but could also be conduction unilever, whatever).